en/flying/basic_flying.md

```diff
@@ -55,6 +53,32 @@ The vehicle may disarm if you take too long to take off after arming (tune the t
 The [Failure Detector](../config/safety.md#failure-detector) will automatically stop the engines if there is a problem on takeoff.
 :::
 
+### Fixed-wing Takeoff
+
+Automatic takeoff using [Takeoff mode (FW)](../flight_modes_fw/takeoff.md) is highly recommended for fixed-wing vehicles!

```

Comment:
I would link the auto takeoff, but recommend that manually piloted flights are also started manually. If the pilot is not capable of a safe takeoff then the landing, which usually is harder, is likely not going to succeed either. Bette then to do the whole flight in a "auto" mindset, with mission takeoff and mission landing.